config/default.yml

```diff
@@ -81,40 +99,48 @@ Layout/MultilineOperationIndentation:
   Description: Checks indentation of binary operations that span more than one line.
   EnforcedStyle: indented
 
-Style/RaiseArgs:
-  EnforcedStyle: compact
-
-Style/SignalException:
-  EnforcedStyle: only_raise
-
-Style/StringLiterals:
-  EnforcedStyle: double_quotes
+#### Rest
+Style/Documentation:
+  Enabled: false
 
 Style/DoubleNegation:
   Enabled: false
 
-Style/RegexpLiteral:
+Style/ExponentialNotation:
+  EnforcedStyle: engineering
```

Comment:
https://docs.rubocop.org/en/latest/cops_style/#styleexponentialnotation `scientific` would be the default value.